lib/recurly/invoice.rb

```diff
@@ -201,6 +202,22 @@ def xml_keys
       super - ['currency']
     end
 
+    # Attempts to update the invoice, returning the success of the request.
+    # Raises an error if attempting to create an invoice using this method.
+    #
+    # @return [true, false]
+    # @raise [RuntimeError] Raises error if you attempt to create an invoice.
+    # @example
+    #   invoice = Recurly::Invoice.find('1000')
+    #   invoice.po_number = '1234'
+    #   invoice.save # => true
+    def save
+      unless persisted?
+        raise "Invoices can only be updated with Invoice#save. New invoices cannot be created using this method."
```

Comment:
Changed error message to clarify that invoices can be updated but not created with this method. Could not use any existing error classes, and it didn't make sense to create a new one at this time.

Reply:
👍